src/components/app/userActionFormVoterRegistration/index.tsx

```
@@ -6,12 +6,12 @@ import {
   SectionNames,
 } from '@/components/app/userActionFormVoterRegistration/constants'
 import { ClaimNft } from '@/components/app/userActionFormVoterRegistration/sections/claimNft'
+import { UserActionFormVoterRegistrationSuccess } from '@/components/app/userActionFormVoterRegistration/sections/success'
 import { Survey } from '@/components/app/userActionFormVoterRegistration/sections/survey'
 import { VoterRegistrationForm } from '@/components/app/userActionFormVoterRegistration/sections/voterRegistrationForm'
 import { useSections } from '@/hooks/useSections'
-import { NFTSlug } from '@/utils/shared/nft'
+import { getValues } from '@/utils/shared/getEntries'
```

I don't think these utils should exists and we shouldn't use them:
1. `Object.values`, `Object.entries` and `Object.keys` already are typesafe
2. The only thing these functions are doing is suppressing errors that we shouldn't want to suppress, this can cause errors impossible to identify bc nothing is being logged
3. Type safety in the reference object will never allow possibly undefined'ish values to be passed to a native call of `Object.values`